src/Bot.hs

```diff
@@ -510,9 +514,11 @@ builtinCommands =
           , nonEmptyRoles
               [qms|You have to be trusted to use this command.
                    Subscribe to gain the trust instantly:
-                   https://twitch.tv/tsoding/subscribe|]
+                   https://twitch.tv/{channel'}/subscribe|]
```

Comment:
Please, get the channel from the Reaction of the command (may require slight modification of nonEmptyRoles Reaction combinator).

Reply:
Thanks, that suggestion cleans things up nicely.